runtimes/python/src/catala/runtime.py

```diff
@@ -416,6 +411,14 @@ def __init__(self, source_position: SourcePosition) -> None:
 # Constructors and conversions
 # ============================
 
+def round(q : Decimal) -> Integer:
```

Comment:
Maybe you could call this function `round_decimal_to_integer`, to not confuse it with `decimal_round`

Reply:
The name is the same in the ocaml runtime, what shall we do?

Reply:
Oh indeed, I let that slip in a previous PR :) You can leave it as `round` I guess.